### ISSUE TITLE
Old browser(s) support

### ISSUE THREAD

**Issue by jakub791** [Patashu/break_eternity.js#114]
Currently break_eternity code (except the typescript version) gets transpiled to ES5 (ES2009) compatible syntax, which I assume is done to keep backwards compatibility. But as far as I know, the only browser that dosen't support ES6 (ES2015) is Internet Explorer (and there was an official statement about the fact that it's no longer gonna be supported IIRC). 
So my question is: do we really need to transpile the code to ES5 compatible syntax?

**Comment by Patashu** [comment]
I think I do this because break_infinity does this because Antimatter Dimensions tried to support Internet Explorer.

Contingent on a convincing argument that this is no longer necessary then I'm happy for it to be changed. Don't know the pros/cons of this decision right now.

**Comment by wearebuilders1233** [comment]
idttiiwibycdwywtdsmh

**Comment by jakub791** [comment]
You shouldn't even think about Internet Explorer.

**Comment by Patashu** [comment]
As long as people still *use* IE11, then surely it's not without value to support it.

**Comment by jakub791** [comment]
If they use 10 years old technologu they should be aware of consequences.

**Comment by jakub791** [comment]
And also: even if you keep supporting IE11, game devs might not do it for the rest of the grand code which defeats the point.

**Comment by jakub791** [comment]
All games that I know use at least some of ES6 syntax which aren't availabe in IE11

**Comment by James103** [comment]
From what I can tell, Microsoft Edge has IE11 compatibility mode; does that limit Javascript to ES5 like in the real IE11?
If so, then this may have to wait until IE11 compatibility mode is depreciated near the end of this decade (at the time of writing).

**Comment by jakub791** [comment]
Tell me one game that is IE11 compatible.

**Comment by jakub791** [comment]
Oh also why is the ESM version of this library transpiled to ES5 syntax? ES modules aren't available in ES5 do.

**Comment by jakub791** [comment]
> As long as people still _use_ IE11, then surely it's not without value to support it.

well ok, I get you. Still would be nice to have a separate modern build of the library for whose who don't need that - surely such code using native classes would run faster than the more verbose ES5 code.
